agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -616,13 +616,15 @@ bool main_thread::backhaul_fsm_main(bool &skip_select)
             FSM_MOVE_STATE(MASTER_DISCOVERY);
         } else { // link establish
 
+#ifdef BEEROCKS_LINUX
```

Comment:
This commit doesn't explain *why* this is done and also why is it done only for Linux.
The commit message should be something like "agent: backhaul: skip removing STA interfaces from the bridge" with a description which explains why.

In addition - the *only* mode is passive mode so we should *never* do any bridge operations. This commit should remove this logic completely.

Reply:
Add a fix up that remove it